session/manager.go

```diff
@@ -51,12 +51,19 @@ type PromiseProcessor interface {
 	Stop() error
 }
 
+// MemorySessionStorage interface to session storage
+type MemorySessionStorage interface {
```

Comment:
Why it's memory? Simply SessionStorage. Structure satisfying this interface and keeping sessions in memory could be called that

Reply:
Since we are in sessions package already. That would give us sessions sessions... 
We could call just "Storage" in that case.

Reply:
I think just `session.Storage` is the best choice.